__tests__/app.test.js

```diff
@@ -31,64 +31,43 @@ describe('app routes', () => {
     });
 
     //TEST FOR COWS DATA
-    test('returns cows', async() => {
-
-      const expectation = [
-        {
-          id: 1,
-          sex: 'male',
-          number_horns: 11,
-          milk: false,
-          cow_breed: 'Sebu',
-        },
-        {
-          id: 2,
-          sex: 'female',
-          number_horns: 22,
-          milk: true,
-          cow_breed: 'charolais',
-        },
-        {
-          id: 3,
-          sex: 'male',
-          number_horns: 22,
-          milk: true,
-          cow_breed: 'Sebu',
-        },
-        {
-          id: 4,
-          sex: 'female',
-          number_horns: 22,
-          milk: false,
-          cow_breed: 'charolais',
-        }
-      ];
+    test('GET/returns cows', async() => {
+
+      const expectedShape =
```

Comment:
its okay to leave your old expectation -- i added this in because i didn't have ids in my expectation but yours was fine the way it was just FYI :)